pulp_container/app/json_schemas.py

```diff
@@ -71,16 +75,7 @@ def get_descriptor_schema(
         "config": get_descriptor_schema([MEDIA_TYPE.CONFIG_BLOB_OCI]),
         "layers": {
             "type": "array",
-            "items": get_descriptor_schema(
-                [
-                    MEDIA_TYPE.REGULAR_BLOB_OCI_TAR,
-                    MEDIA_TYPE.REGULAR_BLOB_OCI_TAR_GZIP,
-                    MEDIA_TYPE.REGULAR_BLOB_OCI_TAR_ZSTD,
-                    MEDIA_TYPE.FOREIGN_BLOB_OCI_TAR,
-                    MEDIA_TYPE.FOREIGN_BLOB_OCI_TAR_GZIP,
-                    MEDIA_TYPE.FOREIGN_BLOB_OCI_TAR_ZSTD,
-                ]
-            ),
+            "items": get_descriptor_schema(),
```

Comment:
from the specs `manifest.layer.mediatype` https://github.com/opencontainers/image-spec/blob/main/manifest.md 
```
Manifests concerned with portability SHOULD use one of the above media types. An encountered mediaType that is unknown to the implementation MUST be ignored.
```